apps/src/lib/ui/checkedRadioButton.css

```diff
@@ -0,0 +1,34 @@
+/*
+Basis for this CSS from:
+https://weblog.west-wind.com/posts/2015/Feb/26/Using-FontAwesome-Fonts-for-HTML-Radio-Buttons-and-Checkboxes
+*/
+
+input[type=radio].with-font {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
```

Comment:
It looks like the height and width you are setting here might make the CheckedRadioButton element tiny, so that when selenium (in UI tests) tries to click on a point in side it, it misses and hits the parent element. to test this hypothesis, you could run the UI test locally with widht and height set to 10px or 20px.

Reply:
another alternative is to use jquery to click the element -- if you go that route, please do a little bit of manual testing to make sure that your CheckedRadioButton really is clickable in all browsers outside of just the 1x1px area.

Reply:
> This seems to be breaking the UI test. I think it's because the input element is hidden and then we add the font awesome icons in its place. Any suggestions on how to re-write the UI test to get around this?

To respond to your comment explicitly, I didn't see a way that the element could be hidden, as opposed to just too small to click. That's a good hypothesis though and if you can click it with jquery but can't click it with selenium when it's 10x10 or 20x20 then I'd believe it is hidden somehow.